Additional text for stop sending variants. (#165)
huitema/dnsoquic: draft-ietf-dprive-dnsoquic.md

In QUIC, sending STOP_SENDING requests that a peer cease transmission on a
stream. If a DoQ client wishes to cancel an outstanding request, it MUST issue
a QUIC STOP_SENDING, and it SHOULD use the error code DOQ_REQUEST_CANCELLED.
It MAY use a more specific error code registered according to {{iana-error-codes}}.
The STOP_SENDING request may be sent at
any time but will have no effect if the server response has already been
sent, in which case the client will simply discard the incoming response.
The corresponding DNS transaction MUST be abandoned.

> Servers that receive STOP_SENDING act in accordance with {{Section 3.5 of RFC9000}}.
> Servers MAY impose implementation limits on the total number or rate of request cancellations.

Reviewer: ```suggestion
Servers that receive STOP_SENDING act in accordance with {{Section 3.5 of RFC9000}}. Servers SHOULD
NOT continue processing a DNS transaction if they receive a STOP_SENDING.

Servers MAY impose implementation limits on the total number or rate of request cancellations.
```

transfer occurring on a given stream without having to close the QUIC
connection.

> Servers MUST NOT continue processing a transaction if they receive a RESET_STREAM

Reviewer: ```suggestion
Servers MUST NOT continue processing a DNS transaction if they receive a RESET_STREAM
```

> request from the client before the client indicates the STREAM FIN. If the
> server has not yet sent the response and indicated the STREAM FIN in the response
> stream or already sent a RESET_STREAM, the server MUST issue a RESET_STREAM to
> indicate that the transaction is abandoned.

Reviewer: ```suggestion
request from the client before the client indicates the STREAM FIN. The server MUST
issue a RESET_STREAM to indicate that the transaction is abandoned unless

* it has already done so for another reason or 
* it has already both sent the response and indicated the STREAM FIN.
```
